src/global.h

```diff
@@ -104,7 +104,7 @@ LED bar:      lbr
 // getting started and software manual URL
 #define CLIENT_GETTING_STARTED_URL       "https://jamulus.io/wiki/Getting-Started"
 #define SERVER_GETTING_STARTED_URL       "https://jamulus.io/wiki/Running-a-Server"
-#define SOFTWARE_MANUAL_URL              "https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md"
+#define SOFTWARE_MANUAL_URL              "https://jamulus.io/wiki/Software-Manual"
```

Comment:
From my POV this should be translatable

Reply:
What do you mean?

Reply:
It should automatically link to the language which is set in Jamulus.

Reply:
Oh ok. Is that something we can do on this PR then?

Reply:
Wouldn't that been done by the Wiki itself?

Reply:
No. The wiki doesn't redirect automatically. Nevertheless, it does show a red message over the navigation if the browser language is available.

Reply:
I'm afraid I'd have no idea how to automate that.

Reply:
I don't even suggest to automate it. We could maybe make the message a bit more prominent.

Reply:
What should happen if there's a translation set for Jamulus but not for that language on the wiki?  Just use the default pages?

I'm pretty sure the path could appear in the translation strings and have a `translate("wiki", url)` wrapper where used to get the translated value.

Reply:
> What should happen if there's a translation set for Jamulus but not for that language on the wiki? Just use the default pages (i.e. same as for any string not translated)?


Yes. I agree.

Reply:
This string should be translatable, then Jamulus GUI translators can do their job when [translating their po files](https://github.com/corrados/jamulus/tree/master/src/res/translation).